src/app_service/service/community/service.nim

```diff
@@ -884,19 +884,17 @@ QtObject:
       if(communitiesSettingsJArr.len == 0):
         raise newException(RpcException, fmt"`communitiesSettings` array is empty in the response for community id: {communityKey}")
 
-      var chatsJArr: JsonNode
-      if(not response.result.getProp("chats", chatsJArr)):
-        raise newException(RpcException, fmt"there is no `chats` key in the response for community id: {communityKey}")
-
```

Comment:
Question:

In https://github.com/status-im/status-go/pull/2694#pullrequestreview-989994420 we expect that there's a `chats` key in the response. Don't we still want to keep this exception then?

Reply:
@PascalPrecht no as I understand we should be able to import community without chats.